docs/Makefile

```diff
@@ -8,7 +9,9 @@ DOCS_ARCHIVE      = public.zip
 help: ## Show this help. Requires locally installed npm and hugo.
 	@awk 'BEGIN {FS = ":.*?## "} /^[a-zA-Z_-]+:.*?## / {sub("\\\\n",sprintf("\n%22c"," "), $$2);printf "\033[36m%-20s\033[0m %s\n", $$1, $$2}' $(MAKEFILE_LIST)
 
-build: submodule ## Generate the static site into the /public folder
```

Comment:
The Hugo build catches it if it is a broken relref link, since Hugo has that functionality built in. But if it's a simple HREF that's broken, it doesn't fail during build but the htmltest fails.